service/azureclusterconfig/v1/key/key.go

```diff
@@ -5,6 +5,11 @@ import (
 	"github.com/giantswarm/microerror"
 )
 
+// ToClusterGuestConfig extracts ClusterGuestConfig from AzureClusterConfig.
+func ToClusterGuestConfig(azureClusterConfig v1alpha1.AzureClusterConfig) v1alpha1.ClusterGuestConfig {
```

Comment:
Nitpick: I don't think it should be prefixed with `To`. There is no type assertion involved.

Reply:
Accepted. I'll change this.